databricks/koalas/base.py

```diff
@@ -323,3 +329,125 @@ def notnull(self):
         return ~self.isnull()
 
     notna = notnull
+
+    # TODO: axis, skipna, and many arguments should be implemented.
+    def all(self, axis: int = 0) -> bool:
+        """
+        Return whether all elements are True.
+
+        Returns True unless there at least one element within a series that is
+        False or equivalent (e.g. zero or empty)
+
+        Parameters
+        ----------
+        axis : {0 or 'index'}, default 0
+            Indicate which axis or axes should be reduced.
+
+            * 0 / 'index' : reduce the index, return a Series whose index is the
+              original column labels.
```

Comment:
nit: if we accept `'index'`, the type hint for `axis` should be `Union[int, str]`?